TestResultSummaryService/parsers/Test.js

```diff
@@ -1,5 +1,5 @@
 const Parser = require( './Parser' );
-const regex = /===============================================\r?\nRunning test (.*?) \.\.\.\r?\n===============================================\r?\n([\S\s]*?\1 Start Time\: .* Epoch Time \(ms\)\: (.*)[\s\S]*?\1_(.*?)\r?\n[\S\s]*?\1 Finish Time\: .* Epoch Time \(ms\)\: (.*))\r?\n/g;
+const regex = /\[.{24}\] ===============================================\r?\n\[.{24}\] Running test (.*?) \.\.\.\r?\n\[.{24}\] ===============================================\r?\n\[.{24}\] ([\S\s]*?\1 Start Time\: .* Epoch Time \(ms\)\: (.*)[\s\S]*?\1_(.*?)\r?\n[\S\s]*?\1 Finish Time\: .* Epoch Time \(ms\)\: (.*))\r?\n/g;

```

Comment:
Could we use `.*`? Matching the exact number of times may fail once we change the format.